packaging/nnstreamer.spec

```diff
@@ -102,7 +104,23 @@ pushd build
 popd
 
 pushd tests
-./testAll.sh
+# The 'testAll.sh' script requires 6~7min to run armv7l binary files in the current CI server.
+# The timeout value is 10min as a heuristic value from our experience.
+timeout=600
+./testAll.sh &
+pid=$!
+while ((timeout > 0)); do
```

Comment:
> What is difference bwteen "(( ....))" and "( ...)" statement?
> And, do we have to use ";" in while ((timeout > 0)); statement?
>

In this case, ( timeout > 0 ) is run 'timeout' with options '>' and '0' and return result of 'timeout' command.
(( timeout > 0 )) is comapre '$timeout' and '0'. if '$timeout' is bigger than 0 return true, else false.

Reply:
got it.